uportal-war/src/main/webapp/media/skins/respondr/common/bootstrap/variables.less

```diff
@@ -14,7 +14,7 @@
 @gray-light:             lighten(@gray-base, 46.7%); // #777
 @gray-lighter:           lighten(@gray-base, 93.5%); // #eee
 
-@brand-primary:         darken(#428bca, 6.5%); // #337ab7
+@brand-primary:         darken(#008bca, 6.5%); // #337ab7
```

Comment:
Agreed with @ChristianMurphy ,
instead of modifying `bootstrap/variables.less` and change `@brand-primary`, you can change the value of the color properties in marketplace.less and use for example the `@greyscaleXX` or `@black` variables of `defaultSkin/less/variable.less` with the right WCAG 2 AA constrast ratio.